pod/cut/templates/video_cut.html

```diff
@@ -0,0 +1,119 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load static %}
+{% load tagging_tags %}
+{% load thumbnail %}
+{% load video_filters %}
+{% load video_tags %}
+{% block page_title %}{% trans 'Cut the video' %} "{{video.title}}" {% endblock page_title %}
+{% block page_extra_head %}
+
+
+{% include 'videos/video-header.html' %}
+
+{% endblock page_extra_head %}
+
+{% block breadcrumbs %}
+{{block.super}}
+<li class="breadcrumb-item"><a href="{% url 'video:my_videos'%}">{% trans 'My videos' %}</a></li>
+<li class="breadcrumb-item">
+  <a href="{% url 'video:video' slug=video.slug %}" title="{{video.title}}">
+    {{video.title|title|truncatechars:45}}
+  </a>
+</li>
+
+<li class="breadcrumb-item active" aria-current="page">        
+    {% trans 'Cut the video' %}
+</li>
+
+{% endblock %}
+{% block page_content %}
+
+<span id="cut_video">
+  {% include 'videos/video-element.html' %}
+</span>
+
+<div id="wrapper_cut">
+  <div id="values_cut">
+    <form action="{% url 'video:cut:video_cut' slug=video.slug %}" method="POST" id="time_for_cut" accept-charset="utf-8" enctype="multipart/form-data">
+      {% csrf_token %}
+
+      <i id="button_start" class="bi bi-scissors" aria-hidden="true" title="{% trans 'Get the video player time' %}"></i>
+      {{ form_cut.video }}
+      {{ form_cut.start }}
+      <i class="bi bi-caret-right-fill"></i>
+      <p id="total_time" title="{% trans 'Total video duration' %}">{{ video.duration_in_time }}</p>
+      <i class="bi bi-caret-left-fill"></i>
+      {{ form_cut.end }}
+      <i id="button_end" class="bi bi-scissors" aria-hidden="true" title="{% trans 'Get the video player time' %}"></i>
```

Comment:
Il me semble que les ciseaux ne sont pas tres parlants pour l'action "Get the player time". Je pense que [box-arrow-down-right](https://icons.getbootstrap.com/icons/box-arrow-down-right/) serait plus parlant.

Reply:
On peut sans doute trouver mieux mais celui-là ne me semble pas plus pertinent

Reply:
Le probleme des ciseaux, c'est qu'on s'attend à avoir une action "couper", qui est plus éloignée de l'action réelle "obtenir le temps". la fleche qui part d'un cadre peut plus facilement faire penser à "obtenir une info" que les ciseaux.

Reply:
Ca me va ;)